pandas/io/xml.py

```diff
@@ -177,7 +177,7 @@ def parse_data(self) -> list[dict[str, str | None]]:
 
         raise AbstractMethodError(self)
 
-    def _parse_nodes(self) -> list[dict[str, str | None]]:
+    def _parse_nodes(self, elems) -> list[dict[str, str | None]]:
```

Comment:
Can `elems` be typed?

Reply:
Yes, it can as a list of parsed elements. So, I will use the list of `Any`.